eng/scripts/vcpkg.ps1

```diff
@@ -0,0 +1,33 @@
+[CmdletBinding()]
+Param (
+    [Parameter()]
+    [ValidateNotNullOrEmpty()]
+    [string] $Ref = (Get-Content "$PSScriptRoot/../vcpkg-commit.txt"),
+
+    [Parameter(Mandatory = $true)]
+    [ValidateNotNullOrEmpty()]
+    [string] $Dependencies,
+
+    [Parameter()]
+    [ValidateNotNullOrEmpty()]
+    [string] $VcpkgPath = "$PSScriptRoot/../../vcpkg"
+)
+
+$initialDirectory = Get-Location
+
+try {
+    git clone https://github.com/Microsoft/vcpkg $VcpkgPath
+    Set-Location $VcpkgPath
+    git fetch --tags
+    git checkout $Ref
```

Comment:
FYI if we ever do put a commit sha in vcpkg-commit.txt, then this checkout might fail if that commit isn't part of the default branch or one of the tags we fetched.

Reply:
Only thought about this because you renamed vcpkg-ref.txt to vcpkg-commit.txt.